config/locales/en.yml

```diff
@@ -359,7 +359,7 @@ en:
       time_based: "Time based: Yes"
     create:
       qrcode_expired: The QR-code and key is expired. Please try registering a new device again.
-      success: You have successfully enabled multi-factor authentication.
+      success: You have successfully enabled OTP based multi-factor authentication.
```

Comment:
Currently, there isn't a flash banner when a TOTP device gets created. The `flash.now[:success] = t(".success")` needs to be a `flash[:success] = t(".success")` in `MultifactorAuthsController#create` as we are redirecting and showing the banner. `.now` banners will be erased when redirected https://api.rubyonrails.org/v6.1.3/classes/ActionDispatch/Flash/FlashHash.html#method-i-now

Can you fix this bug in this PR? Can you also add a controller test to see if the flash banner is shown?

eg. 
```ruby
  should "flash success" do
    assert_equal "You have successfully enabled OTP based multi-factor authentication.", flash[:success]
  end
```